### DISCUSSION TITLE
ClassWiz QR功能分析与内部数据存储原理

### DISCUSSION THREAD

**Discussion by CalcWorld** [Physics365/991CN-X-CW-Decompilation#22] [General]
[ClassWiz QR功能分析与内部数据存储原理.pdf](https://github.com/Physics365/991CN-X-CW-Decompilation/files/14892706/ClassWiz.QR.pdf)
或许可以添加到repo中

**Comment by Physics365** [comment]
教程里后面会加

**Comment by EwTE-H** [comment]
内容太杂了，建议拆分，QR放一段，内部数据放一段，带框字符已经有重复可以不要，都放在不同的.md里面
仓库里面开一个目录专门放cw的文档和教程，格式可以参考现在的cnx教程

**Comment by CalcWorld** [comment]
原文是写在飞书文档的，导出Word后格式比较错乱
下面是节选的数值存储部分，有空再细弄下（
[内部数值存储原理.docx](https://github.com/Physics365/991CN-X-CW-Decompilation/files/14933393/default.docx)
